### PR TITLE
Switch to esbuild

### PR DESCRIPTION
**Pull request by cquiroz**
Turns out that now vite recommends `esbuild` over `terser`. Though it may compress a bit less it is a lot faster and maybe more important the build becomes simpler

Here is a comparison:
ebuild
```
✓ 1461 modules transformed.
../../../../heroku/static/assets/ORCID-iD_icon-vector.15af2c50.svg                               0.94 KiB
../../../../heroku/static/index.html                                                             1.28 KiB
../../../../heroku/static/assets/index.ec417000.css                                              779.85 KiB / gzip: 126.38 KiB
../../../../heroku/static/assets/internal-1589fbf5a2f43f8f14f4bc28a01cf22da04c2328.b1c60ab0.js   647.61 KiB / gzip: 180.28 KiB
../../../../heroku/static/assets/vendor.ac69606c.js                                              1633.70 KiB / gzip: 496.92 KiB
../../../../heroku/static/assets/index.be7653b7.js                                               21754.47 KiB / gzip: 2583.43 KiB
Executed in   36.29 secs    fish           external
   usr time   47.06 secs    0.07 millis   47.06 secs
   sys time    2.66 secs    1.91 millis    2.65 secs
```

terser
```
✓ 1461 modules transformed.
../../../../heroku/static/assets/ORCID-iD_icon-vector.15af2c50.svg                               0.94 KiB
../../../../heroku/static/index.html                                                             1.28 KiB
../../../../heroku/static/assets/index.ec417000.css                                              779.85 KiB / gzip: 126.38 KiB
../../../../heroku/static/assets/internal-1589fbf5a2f43f8f14f4bc28a01cf22da04c2328.af7ec550.js   639.40 KiB / gzip: 174.35 KiB
../../../../heroku/static/assets/vendor.ac69606c.js                                              1602.97 KiB / gzip: 475.43 KiB
../../../../heroku/static/assets/index.bec5d5d7.js                                               21293.10 KiB / gzip: 2359.29 KiB
Executed in  271.82 secs    fish           external
   usr time  321.81 secs    0.07 millis  321.81 secs
   sys time   10.73 secs    2.25 millis   10.73 secs
```

WDYT? Worth the extra size?

### CODE REVIEW

**Comment by rpiaggio** [comment]
It's an order of magnitude faster and the size penalty is only around ~2%.

On the other hand, this is only run when building with fullOpt, right? In that instance we don't have time constraints and can afford it taking longer.

**Comment by cquiroz** [comment]
I think speed is not a relevant factor as we don't do this very often and it is mostly automated

I like the simplification of the build though, as we remove the extra build script'.

I'm not sure though how to interpret the numbers bundlemon gives us

**Review by rpiaggio — APPROVED** [review]
LGTM. Simplifying the build is definitely worth it.

File: package.json [hunk 1]
```diff
@@ -45,7 +45,6 @@
     "@ronilaukkarinen/stylelint-value-no-unknown-custom-properties": "^3.0.1",
     "@vitejs/plugin-react": "^1.2.0",
     "assert": "^2.0.0",
-    "brotli-size": "4.0.0",
     "bundlemon": "^1.3.1",
     "chokidar": "3.5.3",
     "human-format": "0.11.0",
```
Comment (rpiaggio) [inline package.json:50]: I thin `human-format` can also be removed.

Comment (rpiaggio) [reply package.json:50]: Same for `process`.

**Review by rpiaggio — APPROVED** [review]
LGTM

**Pull request merged**